src/components/layout/header/AvatarMenu.tsx

```diff
@@ -100,18 +100,16 @@ const CurrentUserAvatar = forwardRef<HTMLDivElement, CurrentUserAvatarProps>(
     const { fn } = useMantineTheme();
     const border = fn.variant({ variant: 'default' }).border;
 
-    if (!user)
-      return <Avatar ref={ref} styles={{ root: { border: `1px solid ${border}` } }} {...others} />;
-
     return (
       <Avatar
         ref={ref}
-        color={primaryColor}
-        src={getGravatar(user.email)}
-        styles={{ root: { border: `1px solid ${border}` } }}
+        color={user == null ? undefined : primaryColor}
+        src={getAvatarLink(user?.email)}
+        alt={user?.name?.slice(0, 2).toUpperCase() ?? "anon"}
```

Comment:
Why have a differnt alt than the inner content? I'd leave it empty when the name is undefined

Reply:
Alt is not an actual alt, it should always be filled. (I thought it meant alternative image/element too, but it's not it seems)
html Image attribute says it is required, so I just do.